### PR TITLE
Add support for specifying content type

### PR DESCRIPTION
**Pull request by et13**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -28,6 +28,9 @@ inputs:
   tags:
     description: 'Stringified form of a JSON object with the following shape: { [propertyName: string]: string; }'
     required: false
+  contentType:
```
Comment (jimmyca15) [inline action.yml:31]: Is there a version field? If we could bump the minor version that would be great. That way we can signify a new parameter is available.

**Comment by jimmyca15** [comment]
Just curious about the version, everything else looks good.

File: action.yml (hunk 1 above)
Comment (et13) [reply action.yml:31]: Versioning for GitHub Actions uses the tag associated with the GitHub release, and yes, I'll bumping the minor version when I create the new release/tag. If interested, the full versioning details are described [here](https://github.com/actions/toolkit/blob/master/docs/action-versioning.md).

Comment (jimmyca15) [reply action.yml:31]: Sounds great, thanks.

**Pull request merged**